scripts/publish_orb.sh

```diff
@@ -80,17 +80,16 @@ if [ ! -z "$DEV" ]; then
   echo "This will be a dev deployment (prefixed with dev:)"
 fi
 
-
-# Ensure the orb is valid
-./scripts/validate_orb.sh $ORB
-
 ORB_PATH=$(get_orb_path $ORB)
 VERSION=$(get_orb_version $ORB)
 IS_PUBLISHED=$(is_orb_published $ORB)
 IS_CREATED=$(is_orb_created $ORB)
 
+# Ensure the orb is valid
+circleci orb validate $ORB_PATH
```

Comment:
I'm hesitant to not validate canary orbs... It caught some syntax issues for me here https://circleci.com/gh/artsy/orbs/185

Reply:
It still validates them. It just doesn't do the version validation.

Reply:
Ah nice